content/md/supplier-onboarder/use-the-product-grid.md

```diff
@@ -0,0 +1,210 @@
+
+![Browse products' categories](../img/Products_Category1.png)
+
+## View unclassified products
+
+Below the category tree, you will see an additional category: `Unclassified products`. Click on this category to see all unclassified products. This category is added automatically and cannot be removed or edited.
+
+![View unclassified products](../img/Products_Category3.png)
+
+
+## View all products
+
+Click on `All products` to see all products (categorized and uncategorized) of the selected category tree.
+
+![View all products](../img/Products_Category4.png)
+
+## Product counter
+
+Onboarder displays the number of products available in each category and category tree. You will see a number into brackets nearby each category; this figure represents the total number of products classified in this category or sub-category.
+
+At the bottom of the category tree, you will see a Yes/No option: `Include sub-categories`.
+
+![Product counter](../img/Products_Category5.png)
+
+This option takes into account the number of products classified in sub-categories. By default, this option is set to `Yes`; hence sub-categories are included. If you do not want to take into account products classified in sub-categories, select `No`. The tree will then be updated accordingly.
+
+
+## Hide or expand the category panel
+
+To display the product grid widely, you can hide the category panel. To do so, click on the left arrow on the bottom right corner of the panel:
+
+![How to hide the category panel](../img/Hide_category_panel.png)
+
+The category panel will be reduced automatically to the left side.
+
+To expand the panel, click again on the arrow.
+
+![How to expand the category panel](../img/Expand_category_panel.png)
+
+# Customize the grid
+
+## Use filters
+
+The product grid includes filter management.
+
+![Use filters](../img/Products_Filters.png)
+
+You can use the filters displayed or add new filters: click on the `+` icon and select the new attributes you want to use as filters.
+
+If an attribute or an attribute option is not translated in the active locale, its code will be displayed into brackets like [name].
+
+The filters can be applied to the following product information:
+- Product information managed by the Onboarder: Family, Groups, Status, Complete, Creation date, Updated date, Product identifier, Permissions
+- Product information managed via the attributes
+
+
+## Search by label and identifier
+
+To quickly find products, a search bar is available in the product grid.
+The search is only on the identifier (product identifier or product model code) and the label.
+
+![Search bar](../img/Products_search.png)
+
+The behavior of the search is a "contains" filter. For example, if you type in "can", it will display all the products or product models with an identifier/code or a label which contains "can".  
```

Comment:
```suggestion
The search is based on a "contains" filter. For example, if you type in "can", it will display all the products or product models with an identifier/code or a label which contains "can".  
```